### PR TITLE
Redesign according to fiori fundamentals clean

### PR DESCRIPTION
**Pull request by marynaKhromova**
Apply Fiori Fundamentals design and Fiori ngx components for namespaces overview, confirmation modal, create environment modal, environment card component

### CODE REVIEW

**Comment by pekura** [comment]
Two cards on a narrow screen get squeezed unnecessary:

bildschirmfoto 2019-02-20 um 16 29 02

Not sure if we can easily improve it, it is just a minor thing..

**Comment by marynaKhromova** [comment]
> Two cards on a narrow screen get squeezed unnecessary:
> 
> bildschirmfoto 2019-02-20 um 16 29 02
> 
> Not sure if we can easily improve it, it is just a minor thing..

Good point Peter 👍 My idea was to have on small resolutions not 3, but 1 or 2 items by row. But this is a responsive issue.

So as we decided before, we will have a ticket where I will create our own grid and will take care of all responsive issues.

But if you want I can try to make a fix here to prevent squeezing :)

**Comment by pekura** [comment]
If we already have a plan to improve it in the future together with responsivness than I am fine!

File: core/src/app/content/workspace-overview/environment-card/environment-card.component.html [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
-        <div class="sf-sku">
-          <span class="sf-sku__value">{{ entry.services && (entry.services | async) !== null ? (entry.services | async) : 0 }}</span>
-          <span class="sf-sku__label">{{ (entry.services | async) === 1 ? "Service" : "Services" }}</span>
-        </div>
+    <div class="fd-panel__actions">
+      <y-list-actions [entry]="entry" [entryEventHandler]="entryEventHandler" [actions]="actions"></y-list-actions>
+    </div>
+  </div>
+  <div class="fd-panel__body">
+    <div class="fd-container fd-container--fluid">
```
Comment (parostatkiem) [inline core/src/app/content/workspace-overview/environment-card/environment-card.component.html]: I'd love to see `fd-has-padding-tiny` here. I don't like "Show All applications" button touching the bottom.

File: core/src/app/content/workspace-overview/workspace-overview/workspace-overview.component.scss [hunk 2]
```diff
@@ -1,18 +0,0 @@
-.sf {
```
Comment (parostatkiem) [inline core/src/app/content/workspace-overview/workspace-overview/workspace-overview.component.scss]: I think you can remove this file.

File: core/src/styles/fiori-fundamentals-overrides.scss [hunk 3]
```diff
@@ -56,13 +71,13 @@ fd-modal {
 
   &__body {
     &--bleed {
-      
+
       .fd-table {
         td:last-child,
         th:last-child {
           padding-right: 24px;
```
Comment (parostatkiem) [inline core/src/styles/fiori-fundamentals-overrides.scss]: How about some Fundamental variable? 
https://sap.github.io/fundamental/resources/how-to-use-scss.html#core-functions-and-mixins

Comment (marynaKhromova) [reply core/src/styles/fiori-fundamentals-overrides.scss]: As you can see there is no variable for 24px, that's why I used just number. And we need here exactly 24px.

Comment (parostatkiem) [reply core/src/styles/fiori-fundamentals-overrides.scss]: Oh, true

File: core/src/app/content/workspace-overview/workspace-overview/workspace-overview.component.scss (hunk 2 above)
Comment (marynaKhromova) [reply core/src/app/content/workspace-overview/workspace-overview/workspace-overview.component.scss]: done

File: core/src/app/content/workspace-overview/environment-card/environment-card.component.html (hunk 1 above)
Comment (marynaKhromova) [reply core/src/app/content/workspace-overview/environment-card/environment-card.component.html]: `Show All applications` button is not presented on this view.

File: core/src/app/content/environments/environment-create/environment-create.component.css [hunk 4]
```diff
@@ -1,4 +0,0 @@
-.env-create-btn:disabled:hover {
```
Comment (parostatkiem) [inline core/src/app/content/environments/environment-create/environment-create.component.css]: ✂️ 😃

**Pull request merged**